src/modules/footer.js

```diff
@@ -0,0 +1,9 @@
+const footerList = document.querySelector(".footer-ulist");
+const footerLinks = footerList.querySelectorAll("a");
+for (let i = 0; i < footerLinks.length; i++) {
+  footerLinks[i].addEventListener("click", () => {
+    let current = footerList.querySelector(".active");
+    current.className = "footer-a";
+    footerLinks[i].className = "active";
+  });
+}
```

Comment:
```suggestion
const ACTIVE_CLASS = 'active';
const footerLinks = document.querySelectorAll('.footer-ulist .footer-a');
for (const footerLink of footerLinks) {
  footerLink.addEventListener('click', (event) => {
  let active = document.querySelector('.footer-ulist .active');
  active.classList.remove(ACTIVE_CLASS);
    let current = event.target;
    current.classList.add(ACTIVE_CLASS);
  });
}
```

Reply:
Done and  thanks, I didn't like how I had it before and I feel that this way is much better